ToolChain/Assembler/Parser/Parser.py

```diff
@@ -440,19 +348,21 @@ def _evaluateAndExtractInfoFromLine(self, line, buildInstruction):
                     part = part[1:]
                 if not foundSource:
                     foundSource = True
-                    buildInstruction.sourceImmediate = MEMORY_REFERENCE_INDICATORS+ part.upper() + MEMORY_REFERENCE_INDICATORS
+                    buildInstruction.sourceImmediate = ":" + part.upper() + ":"
```

Comment:
Leaving the long form lets us know why specifically `:` is used. While I like it short and sweet, the rest of the code uses the long form and it keeps things orderly.